public/js/charts/worstPMGraph.js

```diff
@@ -122,7 +124,7 @@ const PMTopFiveChart = async (type) => {
   const labels = topFiveNodes.map((_, index) => index + 1);
   let data = Object.values(topFiveNodes).map((worstNodes) => worstNodes[type]);
 
-  data = [...data, AQIBand[type]];
+  data = [...data];
 
   renderWorstNodesChart(labels, data);
```

Comment:
```suggestion
  renderWorstNodesChart(labels, data, type);
```

Use the `type` passed here in the `renderWorstNodesChart` to set the scale instead of `AQIBand.p0`